src/NServiceBus.AcceptanceTests/Core/CoreTestSuiteConstraints.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace NServiceBus.AcceptanceTests.EndpointTemplates
+{
+    public partial class TestSuiteConstraints
```

Comment:
this needs to be implemented by every downstream. The other part of the partial class will throw a compilation error because the interface isn't implemented otherwise.